sparse_list.py

```diff
@@ -125,22 +135,22 @@ def __mul__(self, multiplier):
 
     def count(self, value):
         '''
-        return number of occurrences of value
+        Return the number of occurrences of value.
```

Comment:
I don't recall now, but I *think* I copied the language from the official documentation for `list`...
```sh
> pydoc list | fgrep -A1 count
 |  count(...)
 |      L.count(value) -> integer -- return number of occurrences of value
 |  

> pydoc sparse_list | fgrep -A1 count
     |  count(self, value)
     |      return number of occurrences of value
```